keyboards/lily58/lib/layer_state_reader.c

```diff
@@ -4,10 +4,10 @@
 #include "lily58.h"
 
 #define L_BASE 0
-#define L_LOWER (1 << 1)
+#define L_SUPER (1 << 1)
```

Comment:
please revert these changes. 

If you need the changes, then the contents should be copied to your keymap, instead.

Reply:
hey @drashna I made the suggested changes except for this one. Sorry to ask, but I am curious how/why the keycap specific layer_state_reader should be copied to my keymap. Will a `layer_state_reader.c` file in my keymap dir overwrite the keyboard/lily58 layer_state_reader???? Thanks in advance.

Reply:
No, it wouldn't.  

And honestly, I'm not a fan of the `lib` files, in general.  It's not flexible, and would be better suited for the keymap file, in general.

Reply:
Ok I see. So, I should

1. revert this layer_state_reader
2. move my code from this layer_state_reader to my keycap file

Sound correct?

Thanks again for your guidance. I really appreciate it.

Best,
David

Reply:
@drashna I reverted the layer state reader and put the keycap specific layer state reader code within my keymap. Is that right?

Reply:
Yup, thanks!